### PR TITLE
Revert "Fix up recommendation for finding correct Xcode. "blaze sync --configure" did not appear to work."

### PR DESCRIPTION
**Pull request by brentleyjones**
This reverts commit d99266b6c7c256f330e09157572e544208fce7c4.

The change to the message is only valid for blaze. The previous message is valid for Bazel.

### CODE REVIEW

File: src/main/java/com/google/devtools/build/lib/rules/objc/XcodeConfig.java [hunk 1]
```diff
@@ -397,9 +396,9 @@ private static Map.Entry<XcodeVersionRuleData, Availability> resolveXcodeFromLoc
             String.format(
                 "--xcode_version=%1$s specified, but it is not available locally. Your build will"
                     + " fail if any actions require a local Xcode. If you believe you have '%1$s'"
-                    + " installed, try running \"sudo python3 /usr/local/bin/xcode_configure.py"
-                    + " --verbose\", and then re-run your command. Locally available versions:"
```
Comment (lberki) [inline src/main/java/com/google/devtools/build/lib/rules/objc/XcodeConfig.java:401]: nit: mind fixing the typo ("localy" -> "locally")

Also below and in the test case.

**Review by lberki — APPROVED** [review]

**Comment by brentleyjones** [comment]
@lberki Are you going to manually import this, to keep the blaze specific message set internally (if that's possible)?

**Review by lberki — APPROVED** [review]

**Comment by lberki** [comment]
Argh, I didn't realize Blaze and Bazel need two different messages :( 

I can imagine two approaches (a :

1. Create `AppleSemantics` in the spirit of `CppSemantics` which seems like a cannon a size too big for the task
2. Make this message configurable using a command line flag, which we then duly override at Google

@dmaclach , WDYT?

**Comment by brentleyjones** [comment]
I'm personally not sure, but I feel at a minimum, this should be merged as is, and then from the Blaze side they can figure out the best way. As it is, it's an invalid message for Bazel users.

**Comment by dmaclach** [comment]
Yeah, I guess go ahead with the merge here. Sorry @brentleyjones about any breakage.

@lberki I must admit to not being super excited about either of those options. We don't have any way of discerning between bazel and blaze at runtime?

**Comment by lberki** [comment]
Nope, we don't have a way to distinguish Blaze and Bazel, specifically to prevent people from writing "if Blaze, do X, if Bazel, do Y" shortcuts. @dmaclach I sent you and internal change with the least amount of boilerplate I I think is reasonable to try to address this properly.

**Comment by dmaclach** [comment]
Ack. Thanks @lberki . I'll take a look at the internal CL. In the mean time, LGTM on this PR.

**Comment by lberki** [comment]
With https://github.com/bazelbuild/bazel/commit/b6f55bb99191c6a281d0e0e11ec7a819c6c3ccb7 , I believe this pull request is not necessary anymore: It makes the same change in the error message but leaves it as it is for Google-internal use cases.

**Comment by brentleyjones** [comment]
Thank you!

**Pull request closed without merging**